diff --git a/chrome/android/java/src/org/chromium/chrome/browser/app/ChromeActivity.java b/chrome/android/java/src/org/chromium/chrome/browser/app/ChromeActivity.java
index 9705938266722045fb33f678222f05de2f620091..9a0fcea14fd8b8187ff75b94bd18064cd27fbdeb 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/app/ChromeActivity.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/app/ChromeActivity.java
@@ -655,7 +655,7 @@ public abstract class ChromeActivity extends AsyncInitializationActivity
             mBottomContainer = findViewById(R.id.bottom_container);
 
             mSnackbarManager =
-                    new SnackbarManager(
+                    new org.chromium.chrome.browser.ui.messages.snackbar.BraveSnackbarManager(
                             this,
                             mBottomContainer,
                             getWindowAndroid(),
