diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index 96d41aafb5e6d3eddaeb7eb5f14b7574e2226e4f..ffeffa39d637ae280bec2673b3566f9b93b45d5d 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -4707,6 +4707,7 @@ const FeatureEntry::FeatureVariation
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
