diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index 80bdb78947ad72387d4eeac485766d8d8c9d86c4..64ebba8568960a06526affd7777a38eb75685a31 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -4643,6 +4643,7 @@ const FeatureEntry::FeatureVariation
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
