diff --git a/chrome/browser/ui/tabs/tab_strip_model.h b/chrome/browser/ui/tabs/tab_strip_model.h
index a16dec32ec7c9bec4832816c699d7a6a5a249f36..159a72cef7f55ea099feeebd075f341fa89681b6 100644
--- a/chrome/browser/ui/tabs/tab_strip_model.h
+++ b/chrome/browser/ui/tabs/tab_strip_model.h
@@ -211,6 +211,7 @@ class TabStripModel {
   TabStripModel(const TabStripModel&) = delete;
   TabStripModel& operator=(const TabStripModel&) = delete;
 
+  virtual
   ~TabStripModel();
 
   // Retrieves the TabStripModelDelegate associated with this TabStripModel.
