diff --git a/chrome/browser/ui/views/bookmarks/bookmark_bar_view.cc b/chrome/browser/ui/views/bookmarks/bookmark_bar_view.cc
index 2c7669a6c271db8409736f3b8eedf9e406a93cf5..6bfbb45c2ae7edb4aa6db1a24042b0a561bc56da 100644
--- a/chrome/browser/ui/views/bookmarks/bookmark_bar_view.cc
+++ b/chrome/browser/ui/views/bookmarks/bookmark_bar_view.cc
@@ -894,6 +894,7 @@ void BookmarkBarView::Layout(PassKey) {
       x = next_x;
     }
   }
+  BRAVE_LAYOUT
 
   const bool show_bookmarks_overflow =
       bookmark_service_->loaded() &&
