diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index 9b70ae09c3a8daa27d2ab29d38845f95b8cc07f6..9ffcdb4fb57d48167e7c53bc16eeb9d637243bb7 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2604,6 +2604,7 @@ static_library("browser") {
   } else {
     sources += [ "net/net_error_diagnostics_dialog_stub.cc" ]
   }
+  import("//brave/browser/sources.gni") sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps allow_circular_includes_from += brave_chrome_browser_allow_circular_includes_from
 
   if (is_android) {
     sources += [
@@ -8584,6 +8585,7 @@ static_library("browser_public_dependencies") {
       "//rlz:rlz_lib",
     ]
   }
+  import("//brave/browser/sources.gni") public_deps += brave_chrome_browser_public_deps
 }
 
 # All generated files in //chrome/browser/ depended on by
