diff --git a/net/BUILD.gn b/net/BUILD.gn
index 00ad3e0c6b9f4325ea0eef3ab71ebf6595e10ddc..cc0f051a2641d29bbcba6a80b6d0d93413f09ac2 100644
--- a/net/BUILD.gn
+++ b/net/BUILD.gn
@@ -1280,6 +1280,7 @@ component("net") {
       "reporting/reporting_uploader.h",
     ]
   }
+  sources += brave_net_sources import("//brave/components/brave_wallet/sources.gni") deps += brave_net_deps
 
   if (enable_device_bound_sessions) {
     sources += [
